src/schema.rs

```diff
@@ -177,6 +236,18 @@ pub fn SchemaSection(parquet_info: super::DisplayInfo) -> impl IntoView {
                                         <td class="px-4 py-2 text-gray-500">
                                             {format!("{:.2}%", col.compression_ratio * 100.0)}
                                         </td>
+                                        <td class="px-4 py-2 text-gray-500">{col.null_count}</td>
+                                        <td class="px-4 py-2 text-gray-500">
+                                            <button
+                                                disabled=move || {
+                                                        distinct_values.get().into_iter().filter(|(id,v)| id == &col.id && v != "👁️‍🗨").count() != 0
```

Comment:
Since we are using hash tables here, we can directly call `distinct_values.get().get(id)` to get the values, instead of iterating all values

Reply:
Indeed I've over engineered the solution.